.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -189,6 +189,7 @@ upload_dependencies_arm64:
 upload_system_probe_tests_x64:
   extends:
     - .upload_system_probe_tests
+  rules: !reference [.except_mergequeue]
```

Comment:
Same comment as `.gitlab/functional_test/system_probe.yml`: Does this override inherited rules? Because most/all jobs in this file use `.on_system_probe_changes_or_manual`

Reply:
Addressed in [0afaafc](https://github.com/DataDog/datadog-agent/pull/20520/commits/0afaafcd3095f0e9ed78334e4438a2d7d0ecc46b)